GameServerLib/Logic/Chatbox/Commands/SetCommand.cs

```diff
@@ -22,7 +31,7 @@ public override void Execute(Peer peer, bool hasReceivedArguments, string argume
                 if (int.TryParse(split[2], out fieldNo))
                     if (float.TryParse(split[3], out value))
                     {
-                        //game.GetPeerInfo(peer).Champion.GetStats().setStat((MasterMask)blockNo, (FieldMask)fieldNo, value);
+                        //_game.GetPeerInfo(peer).Champion.GetStats().setStat((MasterMask)blockNo, (FieldMask)fieldNo, value);
```

Comment:
Or fix it, there is no SetStat method in the reworked stats. Who is the author or the rework?